mkdocs-base.yml

```diff
@@ -92,6 +92,7 @@ extra: # Used in main.html template and can't be externalized
 # Common navigation for percona.com, render.com and PDF
 nav:
   - Welcome: index.md
+  - Doc Template: percona-monitoring-and-management-doc-template.md
```

Comment:
I suggest to use some special branch if we would like to have it navigation :)

Reply:
It will not be a part of customer facing document. I will move it to . githib folder after approval.